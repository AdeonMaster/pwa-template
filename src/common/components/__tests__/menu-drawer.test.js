import { render } from '@testing-library/react';

import MenuDrawer from '../menu-drawer';

describe('MenuDrawer', () => {
  it('should render properly', () => {
    const { container } = render(
      <MenuDrawer isOpen={true} toggle={() => undefined}>
        <p>Drawer child content</p>
      </MenuDrawer>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
