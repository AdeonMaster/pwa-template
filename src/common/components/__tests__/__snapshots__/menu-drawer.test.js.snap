// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuDrawer should render properly 1`] = `
<div
  class="drawer-menu right shown"
>
  <div
    class="drawer-menu-backdrop"
  />
  <div
    class="drawer-menu-side align-left"
  >
    <p>
      Drawer child content
    </p>
  </div>
</div>
`;
